### PR TITLE
Rephrase steps in the build jobs, update README.md

### PR DESCRIPTION
**Pull request by raygerlabs**
### Description

This pull request intends to add some minor fixes.

## Proposed Changes
  - Update README.md
  - Re-phrase build and release jobs to have consistent naming.

## Verification

The build jobs shall pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,25 +25,26 @@ The intended usage is subsequent string replacement of project names and setting
 
 ### Build
 
-Conan is introduced as a package manager for managing third-parties but Cmake build is the primary focus.
+Conan is introduced as a package manager for managing third-parties but it is capable of managing whole SW build cycle so effectively replaced Cmake.
```
Comment (raygerlabs) [inline README.md:28]: Rephrase this

File: README.md [hunk 2]
```diff
@@ -25,25 +25,26 @@ The intended usage is subsequent string replacement of project names and setting
...
+# Install project dependencies:
+$ conan install .. -s build_type=Release -pr:b default --build missing
+
+# Build project
+$ conan build .. --configure # only run cmake.configure(). Other methods will do nothing
+$ conan build .. --build     # only run cmake.build(). Other methods will do nothing
+$ conan build .. --install   # only run cmake.install(). Other methods will do nothing
+$ conan build .. --test      # only run cmake.test(). Other methods will do nothing
+# They can be combined
+$ conan build .. -c -b # run cmake.configure() + cmake.build(), but not cmake.install() nor cmake.test
```
Comment (raygerlabs) [inline README.md:46]: Rephrase this or add link to the original conan page

Comment (paulocoutinhox) [reply README.md:46]: Nice tips

File: README.md (hunk 1 above)
Comment (raygerlabs) [reply README.md]: README.md is updated

File: README.md (hunk 2 above)
Comment (raygerlabs) [reply README.md]: Link added here and README.md rephrased

Comment (raygerlabs) [reply README.md]: @paulocoutinhox Yes. It is very convenient. Way more cleaner than the cmake commands.

**Pull request merged**